storm-core/src/ui/public/js/visualization.js

```diff
@@ -127,13 +127,13 @@ function renderGraph(elem) {
                 }
                 
                 var w = Math.max(55, 25 + gfx.textWidth(node.name));
-                
+
```

Comment:
Can we revert this entire file.  No reason to change it just for white space differences.